lib/tire/http/client.rb

```diff
@@ -8,39 +8,39 @@ class RestClient
         ConnectionExceptions = [::RestClient::ServerBrokeConnection, ::RestClient::RequestTimeout]
 
         def self.get(url, data=nil)
-          perform ::RestClient::Request.new(:method => :get, :url => url, :payload => data).execute
+          perform ::RestClient::Request.new(:method => :get, :url => url, :timeout => Configuration.timeout, :open_timeout => Configuration.timeout, :payload => data).execute
```

Comment:
Maybe we should extract a method 
```
def perform_request(method:, url:, data:)
  perform ::RestClient::Request.new(
    :method => method,
    :url => url,
    :timeout => Configuration.timeout,
    :open_timeout => Configuration.timeout,
    :payload => data
  ).execute
end
```

Then we only need to change these in one place going forward.